webpack.config.js

```diff
@@ -1,21 +1,26 @@
-const path = require('path');
 const webpack = require('webpack');
 
-const base = {
+module.exports = {
+    devtool: 'cheap-module-source-map',
+    entry: {
+        'scratch-storage': './src/index-web.js',
+        'scratch-storage.min': './src/index-web.js'
+    },
     module: {
         rules: [
             {
-                include: [
-                    path.resolve(__dirname, 'src')
-                ],
-                test: /\.js$/,
+                test: /[\\/]+scratch-[^\\/]+[\\/]+src[\\/]+.+\.js$/,
```

Comment:
Same comment as on the GUI — I think it would be more understandable if this was more explicit/verbose.